cocos/ui/UIEditBox/UIEditBoxImpl-common.h

```diff
@@ -133,6 +136,14 @@ class EditBoxImplCommon : public EditBoxImpl
     
     int   _maxLength;
     Size _contentSize;
+    
```

Comment:
To use EditBox.h data in EditBox-common.h we need to make it public. Can I make it public?